cmd/eksctl/main.go

```diff
@@ -31,6 +35,17 @@ func addCommands(rootCmd *cobra.Command, flagGrouping *cmdutils.FlagGrouping) {
 }
 
 func main() {
+
+	// Log internally-logged kubernetes errors uniformly
+	logK8sError := func(err error) {
+		// we need to go one level deeper than k8sruntime.HandleError
+		_, file, line, _ := runtime.Caller(1)
+		idx := strings.Index(file, "/k8s.io/")
+		finalFileLine := file[idx+1:] + ":" + strconv.Itoa(line)
+		logger.Warning("Internal Kubernetes error at %s: %s", finalFileLine, err)
```

Comment:
As I mentioned in your other PR, let's not start new lines with capital letters.

Also, it's not internal to Kubernetes itself, it's a client error, I'd make it 'internal Kubernetes client error'.